docs/content/comparisons/druid-vs-sql-on-hadoop.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: doc_page
+---
+
+Druid vs SQL-on-Hadoop (Hive/Impala/Drill/Spark SQL/Presto)
+===========================================================
+
+Druid is much more complementary to SQL-on-Hadoop engines than it is competitive. SQL-on-Hadoop engines provide an 
+execution engine for various data formats and data stores, and 
+many can be made to push down computations down to Druid, while providing a SQL interface to Druid.
+
+For a direct comparison between the technologies and when to only use one or the other, the basically comes down to your 
+product requirements and what the systems were designed to do.  
+
+Druid was designed to
+
+1. be an always on service
+1. ingest data in real-time
+1. handle slice-n-dice style ad-hoc queries
+
+SQL-on-Hadoop engines (as far as we are aware) were designed to replace Hadoop MapReduce with another, faster, query layer 
+that is completely generic and plays well with the other ecosystem of Hadoop technologies.
+
+What does this mean?  We can talk about it in terms of three general areas
+
+1. Queries
+1. Data Ingestion
+1. Query Flexibility
+
+## Queries
+
+Druid segments stores data in a custom column format. Segments are scanned directly as part of queries and each Druid server 
+calculates a set of results that are eventually merged at the Broker level. This means the data that is transferred between servers 
+are queries and results, and all computation is done internally as part of the Druid servers.
+
+Most SQL-on-Hadoop engines are responsible for query planning and execution for underlying storage layers and storage formats. 
+They are processes that stay on even if there is no query running (eliminating the JVM startup costs from Hadoop MapReduce) 
+and they have facilities to cache data locally so that it can be accessed and updated quicker.  
```

Comment:
Is this part "and they have facilities to cache data locally so that it can be accessed and updated quicker" accurate? I didn't think Impala or Presto did that. Can Spark SQL do it with cached RDDs?